vuepress/docs/next/docs/curate/bundle-details.md

```diff
@@ -235,6 +236,48 @@ Depending on the stack type, default values are:
 |`value`|String|No|Value to give to the env variable|
 |`secretKeyRef`|[SecretKeyRef[]](#secretkeyref-specification)|No|A reference to a secret
 
+### Microservices Environment Variables
+|Name| Type | Description| 
+|:-|:-----|:----------------------------------|
+|`KEYCLOAK_REALM`| string | Keycloak or Red Hat Single Sign-On (RH-SSO) realm to be used by the MS | 
+|`KEYCLOAK_AUTH_URL` | string | Keycloak/RH-SSO URL to be used by the MS| 
+|`KEYCLOAK_CLIENT_SECRET`| `secretKeyRef[]`| Keycloak/RH-SSO autogenerated clientSecret to be used by the MS | 
+| `KEYCLOAK_CLIENT_ID`| `secretKeyRef[]`| Keycloak/RH-SSO autogenerated clientId to be used by the MS |
+|`SERVER_SERVLET_CONTEXT_PATH` | string | Context path used to access the MS. Automatically handled by a Spring Boot MS, but can be manually set for other `stack` types|  
+| `SPRING_PROFILES_ACTIVE`| string | Application profile to use when the MS runs on Entando, differentiating dev vs prod at runtime. Automatically handled by a Spring Boot MS but can be manually managed if using another technology `stack` | default, prod |
+| `SPRING_DATASOURCE_URL`| string| Provisioned database JDBC connection URL. Automatically handled by a Spring Boot MS but can be manually managed if using another technology `stack` | 
+| `SPRING_DATASOURCE_USERNAME` | string|  Provisioned database username. Automatically handled for a Spring Boot MS, but can be manually managed if using another technology `stack`| 
+| `SPRING_DATASOURCE_PASSWORD` | string|  Provisioned database password. Automatically handled for a Spring Boot MS, but can be manually managed if using another technology `stack`| 
```

Comment:
general comment wrt `stack`, wondering if backticks are needed? i can see backticking if it's a parameter/property, etc. that can be changed or selected, but "technology stack" (no backticks) is common terminology ??

Reply:
it is a property they need to configure, so it was intentionally highlighted so they'd know they have to set that.